README.md

```diff
@@ -1,111 +1,77 @@
 # David
-An engine for chatbots. 
+
+[![Documentation Status](https://readthedocs.org/projects/david-ausubel/badge/?version=latest)](https://david-ausubel.readthedocs.io/en/latest/?badge=latest)
+
+
+**David** is a powerfull engine dialogue engine with two main parts: `NLU` and `engine`. The main objetive of this project is too use existing chatbots projects and uses it to develop your solution.
+
+The name was inspired by [David Ausubel](https://novaescola.org.br/conteudo/262/david-ausubel-e-a-aprendizagem-significativa) because the main objective of this project was to build a collaborative platform to maintain Bots for learning.
 
 Inspired by Watson Assistant and Rasa.
 
-## How to RUN
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+* Docker:
 
 ```bash
 docker-compose up --build
 ```
 
-or in editable mode:
+* Editable mode:
 
 ```bash
-pip3 install -r requirements-dev.txt
-pip3 install -e .
```

Comment:
In Editable mode, i believe that the initial command must be:

pip install -e .
Because:

pip install -r requirements.txt
Won't make the 'david' command available!

I'm not sure the '-e' option is necessary.

Reply:
I guess using this you install the project locally, so it is good for development. I guess this is the best solution until the package still not published in `pypi`, after that is just needed to change `pip install -e .` to something like `pip install david`.